src/routes/Forms.jsx

```diff
@@ -25,8 +29,29 @@ const Forms = () => {
 
   localStorage.setItem('restorePath', location.pathname);
 
+  const { selectedUser, setSelectedUser } = useContext(SelectUserContext);
+
   return (
-    <>
+    <div
```

Comment:
Change `div` to `Box` and change `style` object to a `sx` object.

Just less MUI refactoring later.